source/s3_auto_ranged_get.c

```diff
@@ -153,25 +137,6 @@ static void s_s3_meta_request_auto_ranged_get_destroy(struct aws_s3_meta_request
     aws_mem_release(meta_request->allocator, auto_ranged_get);
 }
 
-static bool s_s3_auto_ranged_state_has_work(enum aws_s3_auto_ranged_get_state state) {
```

Comment:
Having work is now implicitly defined by the next_request virtual function: if a meta request can return a request for being sent, then it has work.